examples/nodejs-simple/src/index.js

```diff
@@ -17,12 +17,18 @@ const util = require('util');
 
 const sleep = util.promisify(setTimeout);
 
-const agonesSDK = new AgonesSDK();
-
 const DEFAULT_TIMEOUT = 60;
 const MAX_TIMEOUT = 2147483;
 
-const connect = async (timeout) => {
+const connect = async (timeout, enableAlpha) => {
+	let agonesSDK;
+
+	if (enableAlpha) {
```

Comment:
Possible other option - make it look like the Go SDK:
`s.Alpha().SetPlayerCapacity(capacity);` ([source](https://github.com/googleforgames/agones/blob/release-1.6.0/examples/simple-udp/main.go#L346)), where `s` is the SDK instance.

Or would that be a non-canonical Node pattern?

Just wanted to get your thorughts.